server/models/User.js

```diff
@@ -0,0 +1,40 @@
+import { Schema, model } from 'mongoose';
+import { hashSync, genSaltSync, compareSync } from 'bcrypt';
+
+const UserSchema = new Schema({
```

Comment:
Try importing `Schema` from mongoose and using it directly here

Reply:
Thanks, fully noted!